native/src/main/java/io/ballerina/runtime/pipe/utils/Utils.java

```diff
@@ -0,0 +1,28 @@
+package io.ballerina.runtime.pipe.utils;
+
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BError;
+
+import static io.ballerina.runtime.pipe.utils.ModuleUtils.getModule;
+
+/**
+ *  This class contains utility methods for the Pipe module.
+ */
+public class Utils {
+
+    private Utils() {
+
+    }
+
+    // Internal type names
+    public static final String ERROR_TYPE = "Error";
+
+    public static BError createError(String message, String errorTypeName) {
```

Comment:
Since thers's only one error type, we don't need to add the error type as a parameter, right?